### DISCUSSION TITLE
How to format the weekdays?

### DISCUSSION THREAD

**Discussion by 2kjen** [gpbl/react-day-picker#1671] [Support] [answered]
Hi,

I want the weekdays to show 3 characters instead of two. Is there any way to do this in v8? I keep seeing the weekdaysShort prop online, but it seems like this only works in v7.

Also, how can I make the weekdays sentence case (e.g. Mon, Tue, Thur) instead of uppercase?

Thanks!

**Comment by gpbl** [comment, marked as answer]
@2kjen you can pass a `formatWeekdayName` function to the `formatters` prop to format the name of the week:

```tsx
<DayPicker
   formatters={ formatWeekdayName: (day) => { /* return `Mon` or whatever */ }
/>
```

**Reply by 2kjen** [reply]
Thank you!
